### PR TITLE
Fpa

### PR DESCRIPTION
**Pull request by markusschmaus**
@mistercrunch changed rounding of time to work with relative deltas allowing for monthly schedule intervals

### CODE REVIEW

File: airflow/utils.py [hunk 1]
```diff
@@ -502,8 +502,24 @@ def as_tuple(obj):
         return tuple([obj])
 
 
-def round_time(dt, delta):
-    delta = delta.total_seconds()
-    seconds = (dt - dt.min).seconds
-    rounding = (seconds + delta / 2) // delta * delta
-    return dt + timedelta(0, rounding - seconds, -dt.microsecond)
+def round_time(dt, delta, start_date=datetime.min):
```
Comment (mistercrunch) [inline airflow/utils.py:505]: This function is very opaque to me

Comment (markusschmaus) [inline airflow/utils.py:505]: @mistercrunch 
I added comments explaining this function. Does this help?

**Comment by mistercrunch** [comment]
@microblag , can you confirm that this PR doesn't change the current behavior?

**Comment by mistercrunch** [comment]
Merged here:
https://github.com/airbnb/airflow/pull/386

**Pull request closed without merging**